Gemfile

```diff
@@ -22,7 +22,7 @@ end
 # as a dependency of govuk_content_models
 gem 'mongo', '>= 1.7.1'
 
-gem 'gds-sso', '10.0.0'
+gem 'gds-sso', '11.0.0'
```

Comment:
Bumping by a major version normally indicates a breaking change, explicitly so here: https://github.com/alphagov/gds-sso/blob/8f7c32e22202e6ef95dd97f85f2320ffe02d7ab4/CHANGELOG.md.

Is this definitely ok here? What's `gds-sso` used for within this repo?